ansible/roles/products/teleop/server/desktop-icons/templates/scripts/start-docker-container-nuc-haptx.j2

```diff
@@ -11,6 +11,6 @@ docker start ${CONTAINER}
 echo "Starting the container..."
 sleep 3
 docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;roscd sr_ethercat_hand_config/launch;git checkout sr_rhand.launch;git checkout sr_lhand.launch;git checkout ${SR_BRANCH}"
-docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;roslaunch sr_teleop_vive_haptx teleop_vive_haptx_control_right.launch hand_serial:=${HAND_SERIALN} eth_port:=${ETH_PORT} 2> >(tee -a /home/user/.ros/log/stderr.log >&2)"
+docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;ulimit -c unlimited;roslaunch sr_teleop_vive_haptx teleop_vive_haptx_control_right.launch hand_serial:=${HAND_SERIALN} eth_port:=${ETH_PORT} 2> >(tee -a /home/user/.ros/log/stderr.log >&2)"
```

Comment:
also here we don't have anymore this launch file. we should use instead https://github.com/shadow-robot/sr_teleop_vive_haptx_internal/blob/kinetic-devel/sr_teleop_vive_haptx/launch/teleop_vive_haptx_control.launch with side argument